### DISCUSSION TITLE
Commits Work Fine Although Push Contains Colleagues Info 👎

### DISCUSSION THREAD

**Discussion by asos-jackmcalister** [community/community#52175] [New to GitHub]
Hi folks! Hope everyones well! So I've ran into an issue which is really bugging me - I've recently started a new project where I had to reclone a repo, remove original source code and rename it to its new name. Although when I commit a change that works fine and i'm the author ALTHOUGH when I push for some reason it doesn't pick up on my name but a colleague of mine who hasn't worked on this repo. 

- I've checked: "git config --global -l" ----> This shows that I'm the author 
- Also checked: "git config  -l"             ----> This also syncs up with my credentials 

Any help is greatly appreciated!!

**Comment by SMelidoni** [comment]
Hi there @asos-jackmcalister 👋🏽

It seems like you've already checked your local Git configuration. There's a chance that the commit you are pushing might have been authored by your colleague before you cloned the repo. You can check the commit history to see if that's the case.

To resolve the issue, you may want to change the author information of the most recent commit before pushing it. Here's a code block for Github that demonstrates how to do this:

1. Amend the most recent commit to change the author:

`git commit --amend --author="Your Name [your.email@example.com](mailto:your.email@example.com)`

2. Check the commit history to confirm the author has been updated:

`git log -1`

3. If the author information is correct, you can now push the changes:

`git push`

Note: Replace "Your Name" and "[your.email@example.com](mailto:your.email@example.com)" with your actual name and email address.

Hope this helps, please let me know, and I'll be happy to try help you further!
